packages/client/src/graphql/fragments.graphql

```diff
@@ -205,11 +205,18 @@ fragment ProfileStats on ProfileStats {
   mirrors
   quotes
   publications
+  upvotes: reactions(request: { type: UPVOTE })
+  downvotes: reactions(request: { type: DOWNVOTE })
+  upvoted: reacted(request: { type: UPVOTE })
+  downvoted: reacted(request: { type: DOWNVOTE })
+  collects: countOpenActions(request: { anyOf: [{ category: COLLECT }] })
+  countOpenActions(request: $profileStatsCountOpenActionArgs)
```

Comment:
@cesarenaldi I renamed the reactions in the result as requested by a builder in one of our channels, I keep the old naming temporary for compatibility